airbyte-integrations/connectors/source-smartengage/source_smartengage/spec.yaml

```diff
@@ -0,0 +1,19 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/smartengage
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: SmartEngage Spec
+  type: object
+  required:
+    - api_key
+    - avatar_id
+  additionalProperties: true
+  properties:
+    api_key:
+      title: API Key
+      type: string
+      description: API Key
+      airbyte_secret: true
+    avatar_id:
+      title: Avatar ID
+      type: string
+      description: Avatar ID. See <a href="https://smartengage.com/docs/#avatars">here</a>.
```

Comment:
Why not list all avatars (using the endpoint /all_avatar) and use a SubStream to retrieve all data from your account?

Reply:
I was not aware of Substream slicer. Implemented this.